packages/web/docs/package.json

```diff
@@ -16,7 +16,7 @@
     "@radix-ui/react-icons": "1.3.2",
     "@radix-ui/react-tabs": "1.1.2",
     "@radix-ui/react-tooltip": "1.1.6",
-    "@theguild/components": "9.2.0",
+    "@theguild/components": "9.3.1-alpha-20250203132247-81689b6a845e70e06ba48d7ad3c2588bea379e9d",
```

Comment:
Don't merge this PR until https://github.com/the-guild-org/docs/pull/2030 is released and used here.